server.js

```diff
@@ -0,0 +1,16 @@
+console.clear();
```

Comment:
What is the need of console.clear ?

Reply:
It wasn't necessary now. I have used this for clear the terminal for better understanding